charts/harmony-chart/values.yaml

```diff
@@ -364,3 +364,125 @@ velero:
 
 openfaas:
   enabled: false
+
+vector:
+  enabled: false
+  role: "Agent"
+  podDisruptionBudget: # Optional, but recommended
+    enabled: true
+    minAvailable: 1
+  # Configures a PodMonitor CRD used by the Prometheus operator
+  podMonitor:
+    enabled: true
+  tolerations:
+  - operator: Exists
+  logLevel: "info"
+  customConfig:
+    data_dir: /vector-data-dir
+    api:
+      enabled: true
+      address: 0.0.0.0:8686
+      playground: false
+
+    sources:
+      kubernetes_tutor_logs:
+        type: kubernetes_logs
+        extra_namespace_label_selector: app.kubernetes.io/managed-by=tutor
+      kubernetes_global_logs:
+        type: kubernetes_logs
+        extra_namespace_label_selector: app.kubernetes.io/managed-by!=tutor
+        extra_field_selector: |
+          metadata.labels."app.kubernetes.io/name"=ingress-nginx,
+          metadata.labels."app.kubernetes.io/name"=cert-manager
+
+    transforms:
+
+      # Extract logs from Open edX applications
+      application_logs:
+        type: filter
+        inputs:
+        - kubernetes_tutor_logs
+        condition: '!contains(string!(.message), "[tracking]")'
+
+      # Filter out application and global logs whose message is empty to prevent Vector process crash when sending logs to Cloudwatch
+      # More details in https://github.com/vectordotdev/vector/issues/15539
+      typed_application_logs:
+        type: remap
+        inputs:
+        - application_logs
+        source: |-
+          .type = "application"
+        drop_on_error: true
+        drop_on_abort: true
+
+      typed_global_logs:
+        type: remap
+        inputs:
+        - kubernetes_global_logs
+        source: |-
+          .type = "global"
+        drop_on_error: true
+        drop_on_abort: true
+
+      # Appplication logs (OpenedX, ingress-nginx, cert-manager) can be send to cloudwatch
+      # or to s3. It will depend on user needs.
+      non_empty_logs:
+        type: remap
+        inputs:
+        - typed_application_logs
+        - typed_global_logs
+        source: |-
+          if is_empty(string!(.message)) {
+            log("Events with empty message are discarded", level: "info")
+            abort
+          }
+
+      # Extract tracking logs from Open edX applications
+      parsed_tracking_logs:
+        type: remap
+        inputs:
+        - kubernetes_tutor_logs
+        source: |-
```

Comment:
How is this going to work with the existing Aspects vector settings? Are the two going to be exclusive?

Reply:
Yes, both need to be exclusive, for resource consumption and pipeline tracking. However, another filter and a sink can be added to filter specifics of any namespace.

However, keep in mind that Ralph is the recommended pipeline to use.

I can leave examples for it

Reply:
I've updated the values-example.yaml with the elements for one installation filtered by namespace. It just needs to be replicated for every openedx namespace with Aspects installed.